src/glyphs.cpp

```diff
@@ -548,10 +548,6 @@ void RenderSDF(glyph_info &glyph,
         return;
     }
 
-    FT_Glyph ft_glyph = nullptr;
-    ft_glyph_guard glyph_guard(&ft_glyph);
-    if (FT_Get_Glyph(ft_face->glyph, &ft_glyph)) return;
```

Comment:
Seems like it, because the `outline` is available on the ft_face->glyph without `FT_Get_Glyph`